### PR TITLE
Added CFS logic

### PR DESCRIPTION
**Pull request by atul-k-6**
Added python logic for CFS.

### CODE REVIEW

File: cfs.py [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    bs = []
+    with open('output.txt', 'r') as file:
+        for line in file:
+            if line.startswith('pred'):
+                b = float(line.split(':')[1].strip())
+                bs.append(b)
+    # print(bs)
+    # Print the list of values
+    # print(values)
+    ata = [4.00, 2.000, 4.000,3,2,1,4,5]
```
Comment (Sh0g0-1758) [inline cfs.py:172]: why is the ata here random ?

**Review by Sh0g0-1758 — CHANGES_REQUESTED** [review]
hey, can you provide a reason for ata values,

File: cfs.py (hunk 1 above)
Comment (atul-k-6) [reply cfs.py:172]: I just took it from web.

**Review by Sh0g0-1758 — APPROVED** [review]

**Comment by Sh0g0-1758** [comment]
LGTM, thanks for the contribution.

**Pull request merged**